docker-orchestration/osrm-backend/Dockerfile

```diff
@@ -49,6 +49,6 @@ RUN chmod +x /docker-entrypoint.sh
 ARG IMAGE_TAG
 ENV IMAGE_TAG ${IMAGE_TAG}
 
-EXPOSE 5000
+EXPOSE 5000 8090
```

Comment:
Can the oasis service use `5000` too when startup by container? `EXPOSE` two ports normally mean they'll be exposed at the same time.

Reply:
Yes, I think so :)